File: src/main/java/no/nav/fo/veilarbregistrering/arbeidsforhold/ArbeidsforholdGateway.java
package no.nav.fo.veilarbregistrering.arbeidsforhold;

import no.nav.fo.veilarbregistrering.bruker.Foedselsnummer;

public interface ArbeidsforholdGateway {

    FlereArbeidsforhold hentArbeidsforhold(Foedselsnummer fnr);
}

File: src/main/java/no/nav/fo/veilarbregistrering/arbeidsforhold/ArbeidsforholdGateway.kt
package no.nav.fo.veilarbregistrering.arbeidsforhold

import no.nav.fo.veilarbregistrering.bruker.Foedselsnummer

interface ArbeidsforholdGateway {
    fun hentArbeidsforhold(fnr: Foedselsnummer): FlereArbeidsforhold
}